kore/test/Test/Kore/Builtin/Int.hs

```diff
@@ -136,6 +138,18 @@ testComparison symb impl =
   where
     name = expectHook symb
 
+test_reflexivity_symbolic :: TestTree
+test_reflexivity_symbolic =
+    testPropertyWithSolver (Text.unpack name) $ do
```

Comment:
This is an ordinary integration test, not a integration property test. We should use `HUnit` instead, to avoid running the test 100 times.

Please also add a _unit_ (not integration) test to show that `x ==Int y` is inconclusive.